### DISCUSSION TITLE
F43764E4-7E10-4B7F-990C-259DFF63A9BA

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#175] [General]
# F43764E4-7E10-4B7F-990C-259DFF63A9BA

On the second week of May 2024, a series of coronal mass ejections and solar flares originating from a large sunspot group called AR3663, struck the Earth as a powerful geomagnetic storm, with the strongest one so far erupting on May 10th. As a result, it generated aurorae as far south as Florida, USA and as far north as Australia. It is the first G5-class geomagnetic storm observed since 2003 and possibly the strongest one on record since March 1989

https://celestia.mobi/article?guide=F43764E4-7E10-4B7F-990C-259DFF63A9BA

**Comment by kgurtu** [comment]
Magnificent!

**Comment by BBbirdgoogoo** [comment]
cool!

**Reply by BBbirdgoogoo** [reply]
some radiation is beautiful , some radiation is terrible

**Comment by OlivaxNoel** [comment]
COOL

**Comment by gencselim** [comment]
mükemmel bir yaradılış .

**Comment by Bharani528** [comment]
beautiful
